python/src/aac/plugins/generate/generate_impl.py

```diff
@@ -52,7 +53,7 @@ def generate(  # noqa: C901
         )
     except OperationCancelled as e:
         result.status_code = ExecutionStatus.OPERATION_CANCELLED
-        result.add_message(ExecutionMessage(e.message, None, None))
+        result.add_message(ExecutionMessage(e.message, MessageLevel.ERROR, None, None))
```

Comment:
A cancellation wouldn't be an error. There should be an `INFO` level that can be used.